Installation script: compilation failed

I tried to use the installation script `install-lal-and-deps.sh` on Ubuntu 18.04.2 LTS from the current master branch (547daabd5f8392a8dbe93c287f1d79e2dccbddbe).

This results into a compilation error:

```
gprbuild -p -m --target=x86_64-linux  -j0 -XGNATCOLL_MMAP=yes -XGNATCOLL_MADVISE=yes -XGNATCOLL_VERSION=0.0 -XGNATCOLL_OS=unix -XBUILD=PROD  -XLIBRARY_TYPE=static -XXMLADA_BUILD=static -XGPR_BUILD=static \
	-XGNATCOLL_MMAP=yes -XGNATCOLL_MADVISE=yes -XGNATCOLL_VERSION=0.0 -XGNATCOLL_OS=unix -XBUILD=PROD gnatcoll.gpr
Setup
   [mkdir]        object directory for project GnatColl
   [mkdir]        library directory for project GnatColl
Compile
   [C]            update_path.c
   [C]            link_max.c
   [C]            getRSS.c
   [C]            set_std_prefix.c
   [C]            separate_run_path_option.c
   [C]            run_path_option.c
   [C]            gnatcoll_support.c
   [C]            terminals.c
   [C]            objlist_file.c
   [C]            executable_path.c
   [Ada]          gnatcoll-paragraph_filling-words.adb
   [Ada]          gnatcoll-paragraph_filling-badnesses.adb
   [Ada]          gnatcoll-paragraph_filling.adb
   [Ada]          gnatcoll-locks.adb
   [Ada]          gnatcoll-refcount.adb
   [Ada]          gnatcoll-config.adb
gnatcoll-refcount.ads:90:04: warning: in instantiation at gnatcoll-storage_pools-headers.adb:57
gnatcoll-refcount.ads:90:04: warning: possible aliasing problem for type "Header_Access"
gnatcoll-refcount.ads:90:04: warning: use -fno-strict-aliasing switch for references
gnatcoll-refcount.ads:90:04: warning: or use "pragma No_Strict_Aliasing (Header_Access);"
   [Ada]          gnatcoll-pools.adb
   [Ada]          gnatcoll-geometry.adb
   [Ada]          gnatcoll-promises.adb
   [Ada]          gnatcoll-opt_parse.adb
   [Ada]          gnatcoll-templates.adb
   [Ada]          gnatcoll-email-mailboxes.adb
   [Ada]          gnatcoll-utils.adb
gnatcoll-config.ads:63:01: warning: pragma "ADA_05" ignored
   [Ada]          gnatcoll-any_types.adb
   [Ada]          gnatcoll-email-utils.adb
   [Ada]          gnatcoll-remote-db.adb
   [Ada]          gnatcoll-coders-base64.adb
   [Ada]          gnatcoll-os.ads
   [Ada]          gnatcoll-scripts-utils.adb
   [Ada]          gnatcoll-json-utility.adb
   [Ada]          gnatcoll-strings.ads
   [Ada]          gnatcoll-projects-normalize.adb
gnatcoll-email-mailboxes.ads:26:01: warning: pragma "ADA_05" ignored
   [Ada]          gnatcoll-email.adb
gnatcoll-projects-normalize.adb:34:06: warning: license of withed unit "GPR" may be inconsistent
gnatcoll-projects-normalize.adb:35:09: warning: license of withed unit "GPR.Output" may be inconsistent
gnatcoll-projects-normalize.adb:36:09: warning: license of withed unit "GPR.Com" may be inconsistent
gnatcoll-projects-normalize.adb:37:09: warning: license of withed unit "GPR.Tree" may be inconsistent
gnatcoll-projects-normalize.adb:38:09: warning: license of withed unit "GPR.Snames" may be inconsistent
gnatcoll-projects-normalize.adb:39:09: warning: license of withed unit "GPR.Names" may be inconsistent
gnatcoll-projects.adb:11219:41: "Get_Fallback_List" not declared in "Knowledge"
gnatcoll-projects.adb:11224:20: "Free_Knowledge_Base" not declared in "Knowledge"

   compilation of gnatcoll-projects-normalize.adb failed

gprbuild: *** compilation phase failed
Makefile:128: recipe for target 'build-static' failed
make: *** [build-static] Error 4

```

--- a/install-lal-and-deps.sh
+++ b/install-lal-and-deps.sh
@@ -7,12 +7,6 @@ set -e
 # Please see the following section for specific settings you need to do before
 # running it.
 
-##############################################################
-# Change the above with your favorite installation directory #
-##############################################################
-
-INSTALL_DIR=/path/to/install
-
 # Install the following prerequisites:
 #
 # * Python's virtual env;
@@ -22,95 +16,28 @@ INSTALL_DIR=/path/to/install
 #
 #   $ sudo apt-get install python-virtualenv libgmp3-dev
 
-# Install a GNAT Pro or GNAT Community release for your platform in
-# $INSTALL_DIR.  Here, for the example we automatically install GNAT Community
-# 2018 for Linux x86_64 (from https://www.adacore.com/download).
+# You also need to have a GNAT compiler available. Here, for the example we
+# automatically install GNAT Community 2018 for Linux x86_64 (from
+# https://www.adacore.com/download), but a recent FSF release from your
+# favorite distro should do it too.
 
-GNAT_TARBALL=gnat-community-2018-20180528-x86_64-linux-bin
-if ! [ -f "$GNAT_TARBALL" ]
-then
-    wget -O "$GNAT_TARBALL" \
-        http://mirrors.cdn.adacore.com/art/5b0d7bffa3f5d709751e3e04
-fi
-if ! [ -d gnat_community_install_script ]
-then
-    git clone https://github.com/AdaCore/gnat_community_install_script.git
-else
-    (cd gnat_community_install_script && git pull)
-fi
-sh gnat_community_install_script/install_package.sh \
-    ./"$GNAT_TARBALL" "$INSTALL_DIR"
+##################################################
+# Change the below with your desired directories #
+##################################################
 
-# Now, get GNATCOLL sources (core and bindings). This should match the version
-# of the compiler you installed: here, we assume that the "master" branch on
-# GitHub fit with the latest GNAT Community release installed above.
-if ! [ -d gnatcoll-core ]
-then
-    wget http://mirrors.cdn.adacore.com/art/5b0819dfc7a447df26c27a99
-    git clone https://github.com/AdaCore/gnatcoll-core.git
-else
-    (cd gnatcoll-core && git pull)
-fi
-if ! [ -d gnatcoll-bindings ]
-then
-    git clone https://github.com/AdaCore/gnatcoll-bindings.git
-else
-    (cd gnatcoll-bindings && git pull)
-fi
+export ADALIB_DIR=$HOME/adalib
+export TOOLS_DIR=$HOME/build_tools
+export INSTALL_DIR=$HOME/install
+export LIBRARY_PATH=/usr/local/lib
 
-############################################
-# The following should work out of the box #
-############################################
+# This script defers the real work to scripts that are used for travis
+# automated builds. You can see and customize those if you need.
 
-export PATH="$INSTALL_DIR/bin:$PATH"
-export LD_LIBRARY_PATH="$INSTALL_DIR/lib:$LD_LIBRARY_PATH"
+# Change that if you're on a different OS. WARNING: You might have loads of
+# changes to make if you're not under linux.
+export TRAVIS_OS_NAME=linux
+export TRAVIS_REPO_SLUG="AdaCore/libadalang"
+export TRAVIS_BRANCH="master"
 
-# Build GNATCOLL and install it in $INSTALL_DIR
-(
-    cd gnatcoll-core
-    make setup
-    make PROCESSORS=0
-    make prefix="$INSTALL_DIR" install
-)
-(
-    cd gnatcoll-bindings/iconv
-    python2 setup.py build -j0 --prefix="$INSTALL_DIR"
-    python2 setup.py install --prefix="$INSTALL_DIR"
-)
-(
-    cd gnatcoll-bindings/gmp
-    python2 setup.py build -j0 --prefix="$INSTALL_DIR"
-    python2 setup.py install --prefix="$INSTALL_DIR"
-)
-
-# Now prepare the construction of Libadalang itself
-if ! [ -d libadalang ]
-then
-    git clone https://github.com/AdaCore/libadalang.git
-    cd libadalang
-else
-    cd libadalang
-    git pull
-fi
-rm -rf build
-
-# Create a Python virtualenv and install the Libadalang code generation
-# dependencies.
-virtualenv2 lal-venv
-. lal-venv/bin/activate
-pip install -r REQUIREMENTS.dev
-
-# Finally generate Libadalang, build it and install it!
-ada/manage.py make
-ada/manage.py install "$INSTALL_DIR"
-
-# Make the Python binding available
-if [ -z "$PYTHONPATH" ]
-then
-    export PYTHONPATH="$INSTALL_DIR/python"
-else
-    export PYTHONPATH="$INSTALL_DIR/python:$PYTHONPATH"
-fi
-
-# You are ready to run Python scripts using Libadalang!
-python -c 'import libadalang; print libadalang'
+./utils/travis-install.sh
+./utils/travis-script.sh

--- a/utils/travis-install.sh
+++ b/utils/travis-install.sh
@@ -1,5 +1,8 @@
 #! /usr/bin/env sh
 
+# This script is used both as a travis install script, and as a helper to get a
+# quick user/developper Libadalang setup.
+
 set -v
 set -e
 

--- a/utils/travis-langkit-branch.py
+++ b/utils/travis-langkit-branch.py
@@ -18,6 +18,8 @@ pull_request_branch = os.environ.get('TRAVIS_PULL_REQUEST_BRANCH')
 repo_slug = os.environ.get('TRAVIS_REPO_SLUG')
 repo_branch = os.environ.get('TRAVIS_BRANCH')
 
+print("REPO SLUG: ", repo_slug)
+
 
 # Queue of couples (slug, branch) for the Langkit branches to test. The last
 # one is tested first.

